### PR TITLE
Propagate resources to ingest_h5ad node

### PR DESCRIPTION
**Pull request by sgillies**
Take 2: solve the resources propagation problem like VCF does. Instead of #664.

Sorry about the churn @johnkerl, @gspowley, @ktsitsi. How about this less risky approach proven in VCF? I'll make the corresponding PR for our 1-click UI.

### CODE REVIEW

File: src/tiledb/cloud/soma/ingest.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def run_ingest_workflow_udf(
         grf = dag.DAG(
             name=name,
             mode=dag.Mode.BATCH,
-            namespace=namespace,
+            namespace=carry_along.get("namespace", namespace),
```
Comment (sgillies) [inline src/tiledb/cloud/soma/ingest.py:152]: Not related, but clearly missed in the original work.

File: src/tiledb/cloud/soma/ingest.py [hunk 2]
```diff
@@ -132,7 +132,7 @@ def run_ingest_workflow_udf(
             extra_tiledb_config=extra_tiledb_config,
             ingest_mode=ingest_mode,
             platform_config=platform_config,
-            resources=carry_along.get("resources", resources),
+            resources=ingest_resources,
```
Comment (sgillies) [inline src/tiledb/cloud/soma/ingest.py:135]: Here is where we want the UI-selected resources to land and stick.

File: src/tiledb/cloud/soma/ingest.py [hunk 3]
```diff
@@ -418,7 +419,7 @@ def run_ingest_workflow(
         extra_tiledb_config=extra_tiledb_config,
         platform_config=platform_config,
         ingest_mode=ingest_mode,
-        resources=resources,
+        ingest_resources=ingest_resources,
```
Comment (sgillies) [inline src/tiledb/cloud/soma/ingest.py:422]: propagates the UI-selected resources.

**Review by ktsitsi — APPROVED** [review]

**Comment by ktsitsi** [comment]
Will we need a `carry_along` for bioimg too?

File: src/tiledb/cloud/soma/ingest.py (hunk 2 above)
Comment (johnkerl) [reply src/tiledb/cloud/soma/ingest.py:135]: Please add a comment in the code, right here, saying just that. :)

**Review by gspowley — APPROVED** [review]

**Review by johnkerl — APPROVED** [review]

**Comment by sgillies** [comment]
@ktsitsi I honestly don't 100% understand why we're doing `carry_along` here. If you haven't noticed you need it for bio imaging ingestion, then I don't think you need it.

**Pull request merged**